### ISSUE TITLE
Unable to execute script/transaction that takes Type as argument with the flow cli v0.31.2

### ISSUE THREAD

**Issue by onkel-skrue** [onflow/flow-cli#470]
### Problem

Unable to execute script/transaction that takes `Type` as argument with the flow cli v0.31.2 **using emulator**.

### Steps to Reproduce

1. Create`test.cdc` file
```swift
  pub fun main(type: Type): Type {
      return type
  }
```

2. Execute script with flow cli (version v0.31.2)
```
 flow scripts execute ./test.cdc --args-json '[{"type":"Type","value": { "staticType": "Int" }}]'
```

3. Inspect the error
You should get following error:
```
  Command Error: failed to submit executable script: client: rpc error: code = Unknown desc = [Error Code: 1101] cadence runtime error Execution failed:
  error: invalid argument at index 0: cannot import value of type cadence.TypeValue
  --> c8d7f11eb3e66c689948824787ad26617e943407a6ec506b9962bd670ab9ec10
```

**Comment by sideninja** [comment]
Thank you for the feedback, will take a look and report back.

**Comment by onkel-skrue** [comment]
Hello, what is the status on this? Did you find something. Would be awesome if this could get fixed soon.

**Comment by sideninja** [comment]
@onkel-skrue cc-ing @SupunS from Cadence team he might have a better idea what is the problem here.

**Comment by SupunS** [comment]
Hi @onkel-skrue,
I checked the code, and the argument needs to be:
```json
{ "type" : "Type", "value" : { "staticType" : { "kind" : "Int" } } }
```
Seems like documentation (https://docs.onflow.org/cadence/json-cadence-spec/#type) is incorrect there.
We need to update the docs. Thanks for catching that!

**Comment by onkel-skrue** [comment]
Hi @SupunS.

I tried:
```
 '[ { "type" : "Type", "value" : { "staticType" : { "kind" : "Int" } } } ]'
```
but, getting:
```
❌ Command Error: error parsing script arguments: failed to decode value: invalid JSON Cadence structure
```

**Comment by onkel-skrue** [comment]
> Oh, I just realized this feature for importing type values ([onflow/cadence#1202](https://github.com/onflow/cadence/pull/1202)) is not yet released to the networks. This should be available with the next release of Cadence.

Thanks for help. Will wait for it :)

**Comment by onkel-skrue** [comment]
I need a generic script that can check if a collection contains a specific Type of NFT's.

**Comment by onkel-skrue** [comment]
But I don't know what `staticType` really means here...

**Comment by SupunS** [comment]
StaticType is a structure that captures all details about the type.
A sample JSON for a resource would look like: https://github.com/onflow/cadence/blob/314f5fa8a5da8c452d23a42148a1d7927915d613/encoding/json/encoding_test.go#L1180-L1193
Again, I think these should be available in the documentation. I'll create an issue for the doc improvements (Or please feel free to create an issue in https://github.com/onflow/cadence repo).

Looping in @dsainati1 in case I miss anything and/or for any further details

**Comment by sideninja** [comment]
@onkel-skrue so are we okay to close this issue? It looks to me the issues should be opened on Cadence with documentation tag.

**Comment by onkel-skrue** [comment]
Yes, closing now. Thanks for the help :)

**Comment by onkel-skrue** [comment]
```
onkelskrue@Onkels-MBP:~/Projects/foo^master ±
%  flow scripts execute ./test.cdc --args-json '[ { "type" : "Type", "value" : { "staticType" : { "kind" : "Int" } } } ]'


⚠️  Version warning: a new version of Flow CLI is available (v0.33.0).
   Read the installation guide for upgrade instructions: https://docs.onflow.org/flow-cli/install

❌ Command Error: error parsing script arguments: failed to decode value: invalid JSON Cadence structure

onkelskrue@Onkels-MBP:~/Projects/foo^master ±
% flow version                                                                                                    :( 1 22-04-28 - 20:46:02
Version: v0.33.1
```

**Comment by SupunS** [comment]
Unfortunately, this hasn't still made it to the emulator/networks yet. The most recent releases have all been mostly bug fixes, etc. and no major features were shipped. 

More details about the next big release can be found here: https://forum.onflow.org/t/breaking-changes-coming-with-secure-cadence-release/3052, which will also include this feature.
